src/coreclr/jit/emitarm64.cpp

```diff
@@ -8690,19 +8719,20 @@ void emitter::emitIns_R_R_R(
             assert(isFloatReg(reg1));
             assert(isLowPredicateRegister(reg2));
             assert(isVectorRegister(reg3));
-            assert(insOptsScalableWithSimdScalar(opt));
+            assert(insOptsScalable(opt));
+            assert(sopt == INS_SCALABLE_OPTS_NONE);
             fmt = IF_SVE_AK_3A;
             break;
 
         case INS_sve_smaxqv:
         case INS_sve_sminqv:
         case INS_sve_umaxqv:
         case INS_sve_uminqv:
-            unreached(); // TODO-SVE: Not yet supported.
```

Comment:
were you able to verify these?

Reply:
Only that the printing was correct. I'll restore the `unreached()` code